app/pages/settings/settings-diagnostics.tsx

```diff
@@ -17,7 +17,13 @@ export const SettingsDiagnostics = memo(() => {
 
   const updateDiagnosticPermissions = (e: React.ChangeEvent<HTMLInputElement>) => {
     const diagnosticsEnabled = e.target.checked;
-    toast(diagnosticsEnabled ? 'Diagnostics enabled' : 'Diagnostics disabled');
+    toast(
+      <div style={{ textAlign: 'center' }}>
+        {diagnosticsEnabled ? 'Diagnostics enabled' : 'Diagnostics disabled'}
+        <br />
+        (restart needed to apply changes)
```

Comment:
```suggestion
        Restart to apply changes
```

Not big on brackets in UI copy.